packages/template-retail-react-app/app/pages/checkout/partials/contact-info.jsx

```diff
@@ -113,6 +116,89 @@ const ContactInfo = () => {
         }
     }, [showPasswordField])
 
+    const renderLoginState = () => {
```

Comment:
This looks like it could be a good candidate refactor into it's own component so it can be reused. Thoughts?